wlutil/launch.py

```diff
@@ -100,34 +115,56 @@ def launchWorkload(baseConfig, jobs=None, spike=False, interactive=True):
         configs = [baseConfig['jobs'][j] for j in jobs]
 
     baseResDir = wlutil.getOpt('res-dir') / wlutil.getOpt('run-name')
+
+    screenIdentifiers = {}
+
+    try:
+        for config in configs:
+            if config['launch']:
+                runResDir = baseResDir / config['name']
+                uartLog = runResDir / "uartlog"
+                os.makedirs(runResDir)
+
+                if spike:
+                    if 'img' in config and not config['nodisk']:
```

Comment:
You'll have a merge conflict with a recent change I pushed (#224). You can look at the change I made to launch.py there and just apply the same concept here.